### PR TITLE
add comments to config docker target.

### PR DESCRIPTION
**Pull request by morvencao**
resolve: https://github.com/IBM/go-repo-template/issues/51

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -132,6 +132,7 @@ build:
 
 image: build build-push-image
 
+# as an alternative, you can also config docker in your travis.yaml
```
Comment (gyliu513) [inline Makefile]: Can you help add a link of example for reference? @morvencao

Comment (morvencao) [reply Makefile]: I can't find an example.

Comment (gyliu513) [reply Makefile]: If cannot find an example link, can you append an example `travis.yml` in the comment?

**Pull request closed without merging**